src/main/fc/rc_controls.h

```diff
@@ -132,6 +132,7 @@ PG_DECLARE(flight3DConfig_t, flight3DConfig);
 typedef struct armingConfig_s {
     uint8_t gyro_cal_on_first_arm;          // calibrate the gyro right before the first arm
     uint8_t auto_disarm_delay;              // allow automatically disarming multicopters after auto_disarm_delay seconds of zero throttle. Disabled when 0
+    uint8_t prearm_until_first_arm;
```

Comment:
Should we not be using `bool_t`?

Reply:
`gyro_cal_on_first_arm` should actually also be boolean - if needs extended to msp `uint8_t` would be preferred.

Found one instance where boolean is used:

https://github.com/betaflight/betaflight/blob/c0c24d7f524735d73ad11752b483c2f615c72775/src/main/msp/msp.c#L3476

Reply:
> it is a cli value - which does not have a `VAR_BOOL` setting

So, no changes needed? Added those lines by analogy with `gyro_cal_on_first_arm` and others.
Should I "Resolve conversations" and "re-request review"? Sorry for that question, thats my first PR..

Reply:
Just wait for feedback :)